src/test/kotlin/com/mparticle/kits/RoktKitTests.kt

```diff
@@ -264,10 +264,72 @@ class RoktKitTests {
         assertTrue(result.containsKey("key1"))
         assertTrue(result.containsKey("key2"))
         assertTrue(result.containsKey("key3"))
-        assertTrue(result.containsKey("email"))
+        assertTrue(result.containsKey("Email"))
+        assertTrue(result.containsKey("Other"))
+    }
+
+    @Test
+    fun test_addIdentityAttributes_When_userIdentities_Other_map_To_Identity() {
+        val mockFilterUser = mock(FilteredMParticleUser::class.java)
+        val userIdentities = HashMap<IdentityType, String>()
+        userIdentities.put(IdentityType.Email, "TestEmail@gamil.com")
+        userIdentities.put(IdentityType.Other, "hashedEmail@123.com")
+        Mockito.`when`(mockFilterUser.userIdentities).thenReturn(userIdentities)
+        val attributes: Map<String, String> = mapOf(
+            "key1" to "value1",
+            "key2" to "value2",
+            "key3" to "value3",
+        )
+        val hashedField = RoktKit::class.java.getDeclaredField("hashedEmailUserIdentityType")
+        hashedField.isAccessible = true
+        hashedField.set(roktKit, "Other")
+        val method: Method = RoktKit::class.java.getDeclaredMethod(
+            "addIdentityAttributes",
+            Map::class.java,
+            FilteredMParticleUser::class.java,
+        )
+        method.isAccessible = true
+        val result = method.invoke(roktKit, attributes, mockFilterUser) as Map<String, String>
+        assertEquals(5, result.size)
+
+        assertTrue(result.containsKey("key1"))
+        assertTrue(result.containsKey("key2"))
+        assertTrue(result.containsKey("key3"))
+        assertTrue(result.containsKey("Email"))
         assertTrue(result.containsKey("emailsha256"))
     }
 
+    @Test
+    fun test_addIdentityAttributes_When_userIdentities_UNASSIGNED_map_To_Identity() {
+        val mockFilterUser = mock(FilteredMParticleUser::class.java)
+        val userIdentities = HashMap<IdentityType, String>()
+        userIdentities.put(IdentityType.Email, "TestEmail@gamil.com")
+        userIdentities.put(IdentityType.Other, "hashedEmail@123.com")
+        Mockito.`when`(mockFilterUser.userIdentities).thenReturn(userIdentities)
+        val attributes: Map<String, String> = mapOf(
+            "key1" to "value1",
+            "key2" to "value2",
+            "key3" to "value3",
+        )
+        val hashedField = RoktKit::class.java.getDeclaredField("hashedEmailUserIdentityType")
+        hashedField.isAccessible = true
+        hashedField.set(roktKit, "UNASSIGNED")
```

Comment:
this should be `Unknown`

Reply:
https://github.com/mparticle-by-rokt/mPServer/pull/35784/files

I copied from Brian's PR in order to keep it consistent